components/BasicExampleView/BasicExampleView.tsx

```diff
@@ -0,0 +1,189 @@
+import classNames from 'classnames';
+import React, { useCallback, useEffect, useRef, useState } from 'react';
+import { ProjectCode } from '../BasicExampleProjects';
+import { Icon } from '../Icons/Icon';
+import { Sidebar } from './Sidebar';
+import UserContent from './UserContent';
+
+interface DocChangeInfo {
+  type: 'update' | 'initialize' | 'presence';
+  content: string;
+}
+
+interface Props {
+  yorkieClientAddress: string;
+  yorkieDocumentKey: string;
+  yorkieApiKey: string;
+  projectCode: ProjectCode;
+  documentStructure: string;
+  iframeUrl: string;
+}
+
+export const UserNames = {
+  user1: 'User 1',
+  user2: 'User 2',
+  user3: 'User 3',
+  user4: 'User 4',
+};
+export const UserColors = {
+  user1: 'green',
+  user2: 'purple',
+  user3: 'red',
+  user4: 'yellow',
+};
+
+export function BasicExampleView({
+  yorkieClientAddress,
+  yorkieDocumentKey,
+  projectCode,
+  iframeUrl,
+  documentStructure,
+  yorkieApiKey,
+}: Props) {
+  const [docChangeInfos, setDocChangeInfos] = useState<DocChangeInfo[]>([]);
+  const [userList, setUserList] = useState<('user1' | 'user2' | 'user3' | 'user4')[]>(['user1', 'user2']);
+  const [projectCodeState, setProjectCodeState] = useState<ProjectCode>(projectCode);
+  const messagesEndRef = useRef<HTMLDivElement>(null);
+  useEffect(() => {
+    let unsubscribeDoc: Function;
+    let unsubscribePresence: Function;
+    const activate = async () => {
+      const yorkie = await import('yorkie-js-sdk');
+      const client = new yorkie.Client(yorkieClientAddress, {
+        apiKey: yorkieApiKey,
+      });
+      await client.activate();
+      const doc = new yorkie.Document('vuejs-kanban');
+      await client.attach(doc);
+      setDocChangeInfos((prev) => [...prev, { type: 'initialize', content: 'Connection has been established!' }]);
+      unsubscribeDoc = doc.subscribe((event) => {
+        if (event.type === 'remote-change') {
+          for (const changeInfo of event.value) {
+            for (const path of changeInfo.paths) {
+              setDocChangeInfos((prev) => [...prev, { type: 'update', content: path }]);
+            }
+          }
+        }
+      });
+      unsubscribePresence = client.subscribe((event) => {
+        if (event.type === 'peers-changed') {
+          // const documentKey = doc.getKey();
+          // const changedPeers = event.value[documentKey];
+          // setDocChangeInfos((prev) => [...prev, { type: 'presence', content: 'presence change has occurred!' }]);
```

Comment:
Issue2. (yorkie presence) Presence update occurs too frequently

Since some examples will make use of presence update, I have tried to show presence update logs in the Event Log component. However, when I enabled presence update to be pushed to `docChangeInfos`, too many logs appeared. It would be great if somebody could give an advice to how we should manage presence updates in our Event Log component.